### PR TITLE
Remove deprecated ViewPager

### PR DESCRIPTION
**Pull request by diegolmello**

### CODE REVIEW

**Comment by AlexK777** [comment]
closes #1035 

I have made the same changes to my fork and this works as expected on my Pixel 3. @ptomasroos please merge and cut a new release.

**Comment by ptomasroos** [comment]
Thank you! If someone could help review it would be great as I’m busy for two weeks with web projects. 

And again. Big thanks. 

Tomas

**Comment by AlexK777** [comment]
I was able to implement the library with this change into my project and It works as expected on a device with React Native 0.61.1. I was about to submit a PR for the change but then saw this one was already out here. It looks good to me, but the more eyes the better.

**Comment by diegolmello** [comment]
@ptomasroos I think it's safe to merge and release.

**Comment by ParhamZare** [comment]
@ptomasroos Hello please merge this pull request I tested this and working fine .thank you

File: index.js [hunk 1]
```diff
@@ -10,17 +10,17 @@ const {
   ScrollView,
   Platform,
   StyleSheet,
-  ViewPagerAndroid,
   InteractionManager,
 } = ReactNative;
 const TimerMixin = require('react-timer-mixin');
+import ViewPager from "@react-native-community/viewpager";
```
Comment (eXist-FraGGer) [inline index.js]: Please use **require**, **import** could return`undefined` in projects with Typescript

**Comment by heyman333** [comment]
Despite these efforts, I wonder if the maintainers merge this `pr` 

I think this repo is dead ...

File: index.js (hunk 1 above)
Comment (mistikk) [inline index.js]:
```suggestion
const ViewPage = require('@react-native-community/viewpager');
```

**Comment by ptomasroos** [comment]
Waiting for these changes in order to merge.

**Comment by diegolmello** [comment]
@ptomasroos done!

**Comment by ptomasroos** [comment]
Thanks for the help in reviewing and the PR.

**Comment by ChauVV** [comment]
hi @ptomasroos, I's still get this issue on android.

` "react": "16.9.0",
    "react-native": "0.61.2",
    "react-native-scrollable-tab-view": "^1.0.0",`

Screen Shot 2019-10-29 at 11 08 49 AM

**Pull request merged**